src/style.css

```diff
@@ -1,3 +1,73 @@
 body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
   margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+
+  --nav-list-color: #00acdc;
+}
+
+ul,
+li,
+a {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+}
+
+.nav_list ul {
+  display: flex;
+
+  list-style: none;
+  list-style-type: none;
```

Comment:
The `list-style` and `list-style-type` properties are both set to `none`. You only need one of these properties to remove the list styling. Consider removing one to avoid redundancy.